stage-ovos/02-ovos/01-run.sh

```diff
@@ -27,6 +28,18 @@ pip install -r /phal.txt
 pip install -r /speech.txt
 pip install -r /voice.txt
 
+systemctl --user enable ovos
+systemctl --user enable ovos-messagebus
+systemctl --user enable ovos-audio
+systemctl --user enable ovos-voice
+systemctl --user enable ovos-phal
+systemctl --user enable ovos-skills
+systemctl enable ovos-admin-phal
+systemctl enable NetworkManager
+
+mv /root/.config/systemd/user "/home/$FIRST_USER_NAME/.config/systemd/"
+chown -Rv "$FIRST_USER_NAME":"$FIRST_USER_NAME" "/home/$FIRST_USER_NAME/.config/systemd"
+
```

Comment:
I still don't think this is right,  having to do a `mv` shouldn't be necessary.  I feel this is replacing one hack with another

Reply:
We are just moving a symlink. thats all

Reply:
but we shouldn't have to.  I have one working without moving things.  why not put it in the right spot to begin with

Reply:
The correct way would be to login as `ovos` in chroot. `su -c .. ovos` wont budge, as it has problems with `XDG_RUNTIME_DIR`. dont know how to solve that.

Reply:
I think adding th `--login` flag to 'su' solves this
`su -c 'systemctl --user enable ovos.service' --login ovos`
This worked with my current test build

I am currently trying this with NetworkManager also, because that DOES need to start.

Reply:
Just keep in mind that both NetworkManager and admin-phal services need root priviliges